src/storage.rs

```diff
@@ -285,6 +288,33 @@ impl MemStorage {
     pub fn wl(&self) -> RwLockWriteGuard<MemStorageCore> {
         self.core.write().unwrap()
     }
+
+    // actual implementation of the `Storage::entries` method
+    fn entries_impl(&self, low: u64, high: u64, max_size: Option<u64>) -> Result<Vec<Entry>> {
```

Comment:
You don't need to define a new function. Keep it in the original place is OK.

Reply:
fixed